cloudify_hostpool/tests/rest/test_backend.py

```diff
@@ -43,12 +45,16 @@ class RestBackendTest(unittest.TestCase):
 
     def setUp(self):
         pool = {'hosts': self._generate_hosts(self.NUMBER_OF_HOSTS)}
-        self.backend = RestBackend(pool=pool, db_file_name=tempfile.mktemp())
+        fd, self.db_tmp = tempfile.mkstemp()
+        os.close(fd)
```

Comment:
According to tempfile documentation: https://docs.python.org/2/library/tempfile.html the method you've used: tempfile.mktemp() is obsolete and insecure. That's why I used tempfile.mkstemp()